masterlist.yaml

```diff
@@ -789,7 +884,10 @@ plugins:
       - Delev
       - Relev
   - name: 'Hothtrooper44_ArmorCompilation.esp'
-    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/3479/']
+  # Immersive Armors
```

Comment:
No need for the comment since you've added a named link.

Reply:
Also removed now.